backend/src/test/java/com/yagubogu/checkin/service/CheckInServiceTest.java

```diff
@@ -190,4 +190,67 @@ void findCheckInHistory_sortsByCheckInDateDescending() {
         // then
         assertThat(actual.checkInHistory()).containsExactlyElementsOf(expected);
     }
+
+    @DisplayName("직관 인증 내역중 이긴 직관 내역을 모두 조회한다")
+    @Test
+    void findCheckInWinHistory_allCheckInWinsGivenYear() {
+        // given
+        long memberId = 1L;
+        int year = 2025;
+
+        int expectedSize = 5;
+
+        // when
+        CheckInHistoryResponse actual = checkInService.findCheckInWinHistory(memberId, year);
+
+        // then
+        assertThat(actual.checkInHistory().size()).isEqualTo(expectedSize);
+    }
+
+    @DisplayName("직관 인증 내역중 이긴 직관 내역을 인증 날짜 내림차순으로 정렬되어 반환된다")
+    @Test
+    void findCheckInWinHistory_sortsByCheckInDateDescending() {
+        // given
+        long memberId = 1L;
+        int year = 2025;
+
+        List<CheckInGameResponse> expected = List.of(
+                new CheckInGameResponse(1L,
+                        "잠실 야구장",
+                        new CheckInGameTeamResponse(1L, "기아", 10, true),
+                        new CheckInGameTeamResponse(2L, "롯데", 9, false),
+                        LocalDate.of(2025, 7, 21)
+                ),
+                new CheckInGameResponse(3L,
+                        "잠실 야구장",
+                        new CheckInGameTeamResponse(1L, "기아", 10, true),
+                        new CheckInGameTeamResponse(3L, "삼성", 5, false),
+                        LocalDate.of(2025, 7, 19)
+                ),
+                new CheckInGameResponse(4L,
+                        "광주 KIA 챔피언스필드",
+                        new CheckInGameTeamResponse(1L, "기아", 10, true),
+                        new CheckInGameTeamResponse(2L, "롯데", 9, false),
+                        LocalDate.of(2025, 7, 18)
+                ),
+                new CheckInGameResponse(5L,
+                        "광주 KIA 챔피언스필드",
+                        new CheckInGameTeamResponse(3L, "삼성", 1, false),
+                        new CheckInGameTeamResponse(1L, "기아", 9, true),
+                        LocalDate.of(2025, 7, 17)
+                ),
+                new CheckInGameResponse(6L,
+                        "대구 삼성라이온즈파크",
+                        new CheckInGameTeamResponse(1L, "기아", 10, true),
+                        new CheckInGameTeamResponse(2L, "롯데", 9, false),
+                        LocalDate.of(2025, 7, 16)
+                )
+        );
+
+        // when
+        CheckInHistoryResponse actual = checkInService.findCheckInWinHistory(memberId, year);
+
+        // then
+        assertThat(actual.checkInHistory()).containsExactlyElementsOf(expected);
```

Comment:
C: 정렬 여부만 검증하려는 목적이라면, 하드코딩된 리스트 대신 아래처럼 리팩토링해보는 건 어떤가요
현재 방식은 유지보수가 어려워 보이고 코드가 너무 길어집니다
```suggestion
// then
List<LocalDate> sortedDates = new ArrayList<>(actual.checkInHistory());
sortedDates.sort(Comparator.reverseOrder());
    assertThat(actualDates).isEqualTo(sortedDates);
```

Reply:
정렬 여부만 검증하는 목적이 아니라, 이긴 내역만 잘 가져왔는지 값에 대한 검증또한 중요하다고 생각을 해서, containsExactlyElementsOf를 사용했습니다.
다만, DisplayName에 내림차순 정렬만 언급이 되어 있어서 테스트 목적의 오해를 불러 일으킨 것 같네요!
DisplayName을 수정하겠습니다!